Sensus.Android/AndroidSensusServiceHelper.cs

```diff
@@ -833,5 +829,15 @@ public override void BringToForeground()
         }
 
         #endregion
+
+        public void StopAnroidSensusService()
+        {
+            _service.Stop();
+        }
+
+        public SensorManager GetSensorManager()
+        {
+            throw new NotImplementedException();
```

Comment:
Bug, right? Should be implemented as below-left in AnroidSensorListener. I'll fix it.